### DISCUSSION TITLE
로그는 어느 수준까지 찍어야할까요?

### DISCUSSION THREAD

**Discussion by LeeKing2017** [jang-199/DoranDoran#211] [Q&A] [answered]
메소드를 실행 시켰을 때 로그를 찍어서 예기치 못한 에러를 대비하려고 합니다.
그러면 어느 수준까지 로그를 찍어서 이를 해결해야할까요?

**Comment by jang-199** [comment, marked as answer]
info level까지 기록하는 것이 좋겠습니다^&^
